### ISSUE TITLE
[studio] Cluster: When a site created from remote, the remote is only added in the node that created the site.

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#4299]
## Describe the bug
When we have a Studio Cluster and an user creates a site from remote, the remote is only added in the Studio node that created the site, the other node doesn't contain the reference to the remote. 

## To Reproduce
1. Have a Studio cluster
2. Create a site from a Git remote
3. In both nodes check the Site Config - Remote Repositories
4. See that the remote is present in the node that created the site but missing in the other node. 

## Expected behavior
Remotes should be available in all nodes. 

## Screenshots
Node 1 
![image](https://user-images.githubusercontent.com/6722074/93373240-50fbcb00-f812-11ea-8c54-a9d399de6b40.png)

Node 2
![image](https://user-images.githubusercontent.com/6722074/93373263-5a853300-f812-11ea-932a-a6b66ae8e230.png)

## Logs
If the user tries to add the missing remote from the Site Config, Studio will show an error
```
[ERROR] 2020-09-16T17:40:02,346 [http-nio-8080-exec-10] [v2.ExceptionHandlers] | API endpoint http://localhost:8081/studio/api/2/repository/add_remote failed with response: ApiResponse{code=1000, message='Internal system failure', remedialAction='Contact support', documentationUrl=''}                                                                                                                                                                                       │
│ org.springframework.dao.DuplicateKeyException:                                                                                                                                                                                                                                                                                                                                                                                                                                      │
│ ### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: (conn:22) Duplicate entry 'site01node1-origin' for key 'uq_rr_site_remote_name'                                                                                                                                                                                                                                                                                                             │
│ Query is: INSERT INTO remote_repository (site_id, remote_name, remote_url, authentication_type,                                                                                                                                                                                                                                                                                                                                                                                     │
│         remote_username, remote_password, remote_token, remote_private_key)                                                                                                                                                                                                                                                                                                                                                                                                         │
│         VALUES (?, ?, ?, ?, ?,                                                                                                                                                                                                                                                                                                                                                                                                                                                      │
│         ?, ?, ?), parameters ['site01node1','origin','https://github.com/crafter-qa/editorial3110-320assets','none',<null>,<null>,<null>,<null>]                                                                                                                                                                                                                                                                                                                                    │
│ ### The error may involve org.craftercms.studio.api.v2.dal.RemoteRepositoryDAO.insertRemoteRepository-Inline                                                                                                                                                                                                                                                                                                                                                                        │
│ ### The error occurred while setting parameters                                                                                                                                                                                                                                                                                                                                                                                                                                     │
│ ### SQL: INSERT INTO remote_repository (site_id, remote_name, remote_url, authentication_type,         remote_username, remote_password, remote_token, remote_private_key)         VALUES (?, ?, ?, ?, ?,         ?, ?, ?)                                                                                                                                                                                                                                                          │
│ ### Cause: java.sql.SQLIntegrityConstraintViolationException: (conn:22) Duplicate entry 'site01node1-origin' for key 'uq_rr_site_remote_name'                                                                                                                                                                                                                                                                                                                                       │
│ Query is: INSERT INTO remote_repository (site_id, remote_name, remote_url, authentication_type,                                                                                                                                                                                                                                                                                                                                                                                     │
│         remote_username, remote_password, remote_token, remote_private_key)                                                                                                                                                                                                                                                                                                                                                                                                         │
│         VALUES (?, ?, ?, ?, ?,                                                                                                                                                                                                                                                                                                                                                                                                                                                      │
│         ?, ?, ?), parameters ['site01node1','origin','https://github.com/crafter-qa/editorial3110-320assets','none',<null>,<null>,<null>,<null>]                                                                                                                                                                                                                                                                                                                                    │
│ ; SQL []; (conn:22) Duplicate entry 'site01node1-origin' for key 'uq_rr_site_remote_name'                                                                                                                                                                                                                                                                                                                                                                                           │
│ Query is: INSERT INTO remote_repository (site_id, remote_name, remote_url, authentication_type,                                                                                                                                                                                                                                                                                                                                                                                     │
│         remote_username, remote_password, remote_token, remote_private_key)                                                                                                                                                                                                                                                                                                                                                                                                         │
│         VALUES (?, ?, ?, ?, ?,                                                                                                                                                                                                                                                                                                                                                                                                                                                      │
│         ?, ?, ?), parameters ['site01node1','origin','https://github.com/crafter-qa/editorial3110-320assets','none',<null>,<null>,<null>,<null>]; nested exception is java.sql.SQLIntegrityConstraintViolationException: (conn:22) Duplicate entry 'site01node1-origin' for key 'uq_rr_site_remote_name'                                                                                                                                                                            │
│ Query is: INSERT INTO remote_repository (site_id, remote_name, remote_url, authentication_type,                                                                                                                                                                                                                                                                                                                                                                                     │
│         remote_username, remote_password, remote_token, remote_private_key)                                                                                                                                                                                                                                                                                                                                                                                                         │
│         VALUES (?, ?, ?, ?, ?,                                                                                                                                                                                                                                                                                                                                                                                                                                                      │
│         ?, ?, ?), parameters ['site01node1','origin','https://github.com/crafter-qa/editorial3110-320assets','none',<null>,<null>,<null>,<null>]         
```

## Specs
### Version
`3.1.10-SNAPSHOT`

### OS
Linux

### Browser
Any

## Additional context
N/A

**Comment by dejan-brkic** [comment]
Done
PR:
https://github.com/craftercms/studio/pull/2191
https://github.com/craftercms/studio/pull/2192

**Comment by yacdaniel** [comment]
Verified fixed in `3.1.10-SNAPSHOT`

**Comment by yacdaniel** [comment]
Verified.
